### PR TITLE
add org.gnome.design.Contrast

### PR DESCRIPTION
**Pull request by bilelmoussaoui**

### CODE REVIEW

**Comment by bilelmoussaoui** [comment]
bot, build org.gnome.design.Contrast

**Comment by bilelmoussaoui** [comment]
bot, build org.gnome.design.Contrast.

**Comment by bilelmoussaoui** [comment]
bot, build org.gnome.design.Contrast

File: org.gnome.design.Contrast.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "contrast",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=xdg-run/dconf",
```
Comment (x80486) [inline org.gnome.design.Contrast.json]: I think there is an effort going on to remove **the old DConf access method** [here](https://github.com/flathub/flathub/issues/1040) for some applications.

I'm not sure if that applies also to this one (in the future), but it looks like :wink:

Comment (bilelmoussaoui) [reply org.gnome.design.Contrast.json]: I'm aware of that, waiting for 3.34 and a bug on gtk to be fixed as you can't run inspector without having the dconf holes

**Comment by nedrichards** [comment]
Not a blocker, but when I open the about dialog I get:
> (contrast:3): Gtk-WARNING **: 21:45:34.228: Could not load a pixbuf from icon theme.
This may indicate that pixbuf loaders or the mime database could not be found.

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/org.gnome.design.Contrast

**Comment by bilelmoussaoui** [comment]
@nedrichards thanks!! Will track that for the next release :)

**Pull request closed without merging**